test/test_http.rb

```diff
@@ -84,7 +84,7 @@ def test_terminates_on_timeout
       end
       sleep 0.25
       res = Zold::Http.new(uri: "http://127.0.0.1:#{port}/").get(timeout: 0.1)
-      assert_equal(599, res.status, res)
```

Comment:
@dapi this is a serious change in the protocol. We have to get 599 in case of failures. Zero is not an acceptable output.

Reply:
ok, I get

Reply:
thx

Reply:
@yegor256 why used this solution? Why does not compare `res` class with `Error` class? Why it does not raise and exception?

Reply:
I fix that. There are now dependencies on 599 except tests:

```
lib/zold/http.rb:        code == 0 ? 599 : code
lib/zold/http.rb:        599
test/test_http.rb:    assert_equal(599, res.status)
test/test_http.rb:      assert_equal(599, res.status, res)
test/node/fake_node.rb:          break unless ping.status == 599 && node.alive?
```

I think use fictional http status is not good idea. Good:

```
res = http.get()
if res.is_a?(Error)
  do_some()
end
```

Better - raise exception and catch it when you need.